Module_Postman/activities/Act7_API_Documentation_in_Postman/2.md

```diff
@@ -0,0 +1,15 @@
+<!--title={API Documentation and Communication}-->
+
+<!--badges={Web Development:}-->
+
+<!--conecpts={}-->
+
+More often than not, as a developer, one will be placed in a team of other developers. This in turn can lead to some misunderstandings and confusions when developing side by side with others as it can be difficult to understand other people's code at times.
```

Comment:
Might be worth mentioning you could be working on an API intended for public consumption in which case the documentation can play a role in adoption and onboarding (which may be business goals).